app/src/main/res/layout/fragment_main.xml

```diff
@@ -23,6 +24,13 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            android:text="Hello test of hardcoded string"/>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/res/layout/fragment_main.xml#L32"></span>Hardcoded string “Hello test of hardcoded string”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>